example/lib/navigation/navigator_bloc.dart

```diff
@@ -0,0 +1,26 @@
+import 'dart:async';
+import 'package:blemulator_example/navigation/route_name.dart';
+import 'package:bloc/bloc.dart';
+import 'package:flutter/material.dart';
+import './bloc.dart';
+
+class NavigatorBloc extends Bloc<NavigationEvent, dynamic> {
+  final GlobalKey<NavigatorState> navigatorKey;
+
+  NavigatorBloc({this.navigatorKey});
+
+  @override
+  dynamic get initialState => 0;
```

Comment:
Why dynamic?

Reply:
Changed type to void. Renamed NavigatorBloc to NavigationBloc.